starter-code/pizza.js

```diff
@@ -1 +1,64 @@
 // Write your Pizza Builder JavaScript in this file.
+
+'use strict'
+
+function main(){
+
+    var sauce = '$0 regular sauce';
+    var crust = '$0 regular crust';
+
+    var $peppers = $('.green-pepper');
+    var $peppersButton = $('.btn-green-peppers');
+    var $pepperList = $('.price li:nth-child(1)');
+        $peppersButton.click(function(){
```

Comment:
Here you change the visibility of the peppers, but you don't update the price. Same happens with pepperoni and mushroms. However, with sauce and crust you don't change the visibility of the elements, but you update the prices. Then, I understand that you know how to do the two thing but you choose to do only one every time. Why?